runestone/common/project_template/conf.tmpl

```diff
@@ -92,10 +92,20 @@ pygments_style = 'sphinx'
 
 # `keep_warnings <http://www.sphinx-doc.org/en/stable/config.html#confval-keep_warnings>`_:
 # If true, keep warnings as “system message” paragraphs in the built documents.
-# Regardless of this setting, warnings are always written to the standard error 
+# Regardless of this setting, warnings are always written to the standard error
 # stream when sphinx-build is run.
 keep_warnings = True
 
+# `rst_prolog <http://www.sphinx-doc.org/en/stable/config.html#confval-rst_prolog>`_:
+# A string of reStructuredText that will be included at the beginning of every
+# source file that is read.
+rst_prolog = (
+# For fill-in-the-blank questions, provide a convenient means to indicate a blank.
```

Comment:
Yes, that's correct; any form of ``:blank:`foo` `` (where `foo` can be anything) works.. Would you like me to submit PRs for these books? Or would you prefer to edit them yourself?